### DISCUSSION TITLE
Are image classification models supported by Frigate?

### DISCUSSION THREAD

**Discussion by hursey013** [blakeblackshear/frigate#586] [Ask A Question]
First off, thank you for all of your great work on this project. As a complete noob to object detection I was able to follow the great documentation and get up to speed in no time.

I'm looking forward to models of different sizes being supported through https://github.com/blakeblackshear/frigate/pull/329/commits/91cde6ce7be572224d425da398ff3b325975b0d9 because I am hoping to use the MobileNet V2 (iNaturist birds) model, but now realize that it is an image classification model rather than an object detection model.

Without knowing much about the underlying differences between the model types, would an image classification model be able to be used with Frigate out of the box, or would there be an intermediate step before being able to use it?

**Comment by blakeblackshear** [comment]
It won't work. The input and output shapes are totally different.

**Reply by hursey013** [reply]
Bummer, appreciate the prompt reply.
